dockerfiles/platform-gnn/Dockerfile

```diff
@@ -20,11 +20,10 @@
 # Others may not be wanted at all in container mode - tensorflow, etc  ??
 
 ARG BUILD_LABEL
-ARG DOCKER_REPO
 FROM ${DOCKER_REPO}/platform-core:${BUILD_LABEL}
```

Comment:
Do we need to have 
ARG DOCKER_REPO

before using the variable here?

Reply:
fixed